### DISCUSSION TITLE
Only flatpaks now?

### DISCUSSION THREAD

**Discussion by elSolus** [elementary/appcenter#1602] [Q&A] [answered]
Hi. I see only flatpaks in the AppCenter now. Where are the apps that were available in there before, in Hera? I mean I really miss "Monitor" by Stanislaw Dac. And many other apps from Hera-era are just gone. Almost all apps now give non-curated warning.

**Comment by cassidyjames** [comment, marked as answer]
AppCenter app developers are porting their apps to Flatpak and OS 6; we started talking to devs about it months ago (during the beta process) but some decided to wait until after the release to port their apps. You can follow along with the progress of those developers by watching [appcenter.elementary.io](https://appcenter.elementary.io) and seeing the apps that are at the top; they're all ported to Flatpak and available now in OS 6.

Once huge benefit to the transition to Flatpak is that these apps will generally work in perpetuity; they won't need to be ported to run on elementary OS 7 and beyond in the future.

If an app is showing as non-curated, it’s from a third-party source like Flathub.
